src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/FocusableLabelTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace System.Windows.Forms;
+
+public class FocusableLabelTests
+{
+    [Fact]
+    public void UnderlineWhenFocused_DefaultValue_ShouldBeTrue()
+    {
+        using FocusableLabel label = new();
+
+        label.UnderlineWhenFocused.Should().BeTrue();
+    }
+
+    [Fact]
+    public void FocusableLabel_ShouldBeSelectableAndTabStop()
```

Comment:
```suggestion
    public void TabStop_ShouldBeTrue()
```
We are not testing `Selectable` here.